tests/integration/x-product-test.js

```diff
@@ -1,21 +1,21 @@
 import hbs from 'htmlbars-inline-precompile';
 import { moduleForComponent, test } from 'ember-qunit';
-import initializer from '../../initializers/ember-intl';
+import instanceInitializer from '../../instance-initializers/ember-intl';
```

Comment:
I think, it'll work 
import instanceInitializer from '../../../instance-initializers/ember-intl';